.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will install RST and eventually run tests
+
```

Comment:
Are you able to add your authorship to this file @aburrell?  I see @mts299 and I did this for the mkdocs.yml file on the top directory.

Reply:
Is there another place I should put authorship?  Do you want it on this file in particular?